Scala/Scala.sublime-syntax

```diff
@@ -138,7 +138,7 @@ contexts:
           pop: true
         - match: '([a-zA-Z$_][a-zA-Z0-9$_]*)\s*:\s*'
           captures:
-            1: entity.name.variable variable.parameter
+            1: entity.name.parameter variable.parameter
```

Comment:
However, for code highlighting, it seems it would be better to leave it as `variable.parameter` and instead add a `.tmPreferences` file that indicates `variable.parameter` should be indexed.